src/appmixer/utils/csv/AddRow/AddRow.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
             parseNumbers,
             parseBooleans
         });
-
+        const lock = await context.lock(fileId, { retryDelay: 1000 });
```

Comment:
There has to be the try-finally block as if any of the code below fails and throws an exception, the `lock` remains locked, right?